phosphord/README.md

```diff
@@ -0,0 +1,28 @@
+# PhosphorD
+
+PhosphorD is a local forwarder, like StatsD, which receives traces from the Phosphor client, and forwards to the [Phosphor server](https://github.com/mondough/phosphor/tree/master/phosphor).
+
+Currently this receives Traces over UDP, which prevents clients blocking, but is reasonably reliable on a local machine. In the event this blocks, traces will be dropped and lost.
+
+A future improvement would make this configurable to read from local files, mirroring the behaviour of Dapper Daemons as described in the [Google Dapper](https://research.google.com/pubs/pub36356.html) paper.
```

Comment:
Not a point for now of course, but using a domain socket in `SOCK_DGRAM ` mode would seem to offer the best of both worlds (obviously at the expense of Windows support, if that's needed/wanted).